spring-kafka-test/src/main/java/org/springframework/kafka/test/utils/KafkaTestUtils.java

```diff
@@ -108,30 +108,46 @@ private KafkaTestUtils() {
 	}
 
 	/**
+	 * Same as {@link #getSingleRecord(Consumer, String, long)} with a timeout of 60 seconds.
```

Comment:
Well, JavaDocs are not arbitrary text.
We don't tell `Same as` from the beginning.
Would be better to copy/paste JavaDocs from existing methods and add `@see` in the and to refer user to those siblings.
And `@since 2.0` to the new methods.

Plus you name, please, into the `@author` list.